### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by atuchin-m**

### CODE REVIEW

**Review by linhkikuchi — DISMISSED** [review]

**Comment by rebron** [comment]
@atuchin-m LGTM ++

File: README.md [hunk 1]
```diff
@@ -1,17 +1,18 @@
-# Translation relay server for brave
+# Translation server for brave
 
-`go-translate` implements a translation relay server for use in brave-core written in Go.
+`go-translate` implements a translation server for use in brave-core written in Go.
+It works on top of brave-hosted [Lingvanex On-premise Translation Server](https://lingvanex.com/translationserver/). It gets the requests from browsers in Chromium format,
+rewrite them to Lingvanex format, process requests and returns the result back to Chromium format.
```
Comment (iefremov) [inline README.md:5]: rewrites, processes the requests
back in Chromium format

File: README.md [hunk 2]
```diff
@@ -38,3 +39,26 @@ make build
...
 `make build`
+
+## Local debugging
+
+- Generate TLS credentials:
+`openssl genrsa -out server.key 2048`
+`openssl ecparam -genkey -name secp384r1 -out server.key`
+`openssl req -new -x509 -sha256 -key server.key -out server.crt -days 3650`
+
+- Replace `err := srv.ListenAndServe()` to `err := srv.ListenAndServeTLS("server.crt", "server.key")` in `server/server.go`;
```
Comment (iefremov) [inline README.md:50]: Replace with

File: README.md [hunk 3]
```diff
@@ -38,3 +39,26 @@ make build
...
+- Launch the local server: `make build`;
+
+- Launch the browser with switches:
+`--ignore-certificate-errors --translate-security-origin=https://127.0.0.1:8195/ --translate-script-url=https://127.0.0.1:8195/static/v1/element.js`.
+
+- Other switches can be added if necessary (for example `--enable-features=UseBraveTranslateGo:update-languages/true` );
+
+- Disable Shield on the tested sites (or globally), it cuts requests to localhost. `405 OPTIONS` errors also can be ignored;
+
+- If you have troubles check you can reach https://127.0.0.1:8195/static/v1/element.js in the tested browser.
```
Comment (iefremov) [inline README.md:64]: check if you can

**Review by iefremov — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (atuchin-m) [reply README.md:5]:
```suggestion
rewrites them to Lingvanex format, processes the requests and returns the result back in Chromium format.
```

File: README.md (hunk 2 above)
Comment (atuchin-m) [reply README.md:50]:
```suggestion
- Replace `err := srv.ListenAndServe()` with `err := srv.ListenAndServeTLS("server.crt", "server.key")` in `server/server.go`;
```

File: README.md (hunk 3 above)
Comment (atuchin-m) [reply README.md:64]:
```suggestion
- If you have troubles check if you can reach https://127.0.0.1:8195/static/v1/element.js in the tested browser.
```

**Review by iefremov — APPROVED** [review]

**Review by linhkikuchi — APPROVED** [review]

**Pull request merged**